docs/stencil-docs/storefront-customization/theme-accessibility.md

```diff
@@ -19,36 +19,25 @@ We have included four accessibility best practices and examples that will help y
 * A store theme. (The following examples use the Cornerstone theme.)
 
 ## Image alt text
-Alternative (alt) text is an image description read aloud by a screen reader. Adding alt text increases the content's accessibility to visually-impaired users. All HTML `<img>` elements should use the `alt` attribute to provide a text description of the image.
+Alternative (alt) text is an image description read aloud by a screen reader. Adding alt text increases the content's accessibility to visually-impaired users. All HTML `<img>` elements should use the `<alt>` attribute to provide a text description of the image.
 
-**Store logo example**
+**Alt image text using HTML**
 
-In Cornerstone, go to `templates/components/common/store-logo.html`. You will see the following code:
-
-```html
- <a href="{{urls.home}}" class="header-logo__link">
-    {{#if settings.store_logo.image}}
-        {{#if theme_settings.logo_size '===' 'original'}}
-            <img class="header-logo-image-unknown-size" src="{{getImage settings.store_logo.image 'logo_size'}}" alt="{{settings.store_logo.title}}" title="{{settings.store_logo.title}}">
-        {{else}}
-            <div class="header-logo-image-container">
-                <img class="header-logo-image" src="{{getImage settings.store_logo.image 'logo_size'}}" alt="{{settings.store_logo.title}}" title="{{settings.store_logo.title}}">
-            </div>
-        {{/if}}
-    {{else}}
-        <span class="header-logo-text">{{settings.store_logo.title}}</span>
-    {{/if}}
-</a>
-```
-Add or update the image alt text to provide descriptive text for your store logo.
+In Cornerstone, go to `templates/components/common/store-logo.html`. In the `<img>` tag, provide descriptive alt text for your store logo.
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+ 
+NOTE: You can update the `<title>` tag with the same text as the `<alt>` tag to create a tooltip, or remove it if you do not need it.
+</div>
+</div>
+</div>
 
 ```html
-alt = "BigCommerce store logo" 
+<img class="header-logo-image" src="https://cdn11.bigcommerce.com/{{store-domain}}/images/stencil/250x100/4f4f5910-4185-4897-9648-0b841b36d530_4_5005_c_1591994810__53965.original.jpeg 'logo_size'}}" alt="BigCommerce store logo" title="BigCommerce store logo"> 
```

Comment:
Shortening this example to remove unnecessary attributes in the `<img>` tag and provide a less complex image source example
```suggestion
<img src="store_logo.jpeg" alt="BigCommerce store logo"> 
```